lbrynet/daemon/testMedia.py

```diff
@@ -0,0 +1,3 @@
+from MediaDescriptor import get_video_length
+
```

Comment:
what is this for?

Reply:
that is is a big oops. It shouldn't be there. Sorry.